environment.yml

```diff
@@ -3,7 +3,11 @@ channels:
   - conda-forge
   - bioconda
 dependencies:
-  - python==3.8
+  - gcc
+  - gmp
+  - gxx
+  - mpfr
+  - python
```

Comment:
don't think you need python here as a dependency

Reply:
I removed python